src/main.js

```diff
@@ -9,19 +9,27 @@ var unfavoriteStar = document.querySelectorAll(".star-inactive");
 
 saveButton.disabled = true; //need to add class in CSS to change appearance
 saveButton.addEventListener("click", function(){
-  createIdeas();
-  displayIdeas();
+  var currentIdea = createIdeas();
+  displayIdeas(currentIdea);
   clearFields();
 });
+window.addEventListener('load', function(){
```

Comment:
I think I understand the code here, but I want to discuss it together after tomorrow's class. I think I'll have a better understanding after that lesson and be able to use better terms of art with a more accurate explanation.